messages/en.json

```diff
@@ -2,12 +2,49 @@
   "title": "MABP Brand Website EN",
   "description": "Best Brand Website in the world!",
 
-  "Index": {
-    "title": "Hello world!"
-  },
   "navbar": {
     "home": "Home",
     "aboutus": "About us",
     "blog": "Blog"
+  },
+  "Header": {
+    "title": "From stellar websites to unique products, we deliver ",
+    "accent_title": "out-of-this-world solutions",
+    "contact_us": "Contact us",
+    "our_services": "Our services",
+    "subtitle_one": "Whether it is a showcase website, MVP, or a complex project,",
+    "subtitle_accent": " we oversee the entire process. ",
+    "subtitle_two": "Count on us to turn your ideas into solutions—where engineering excellence meets innovation.",
+    "subtitle_three": "Your journey begins here, where we bring your vision to life"
+  },
+  "Services": {
+    "servicesTitle": "More than building websites",
+    "consultancyTitle": "Consultancy",
```

Comment:
I think here you could use a more structured way. For example: having an "Consultancy" property and inside of it having the title, the description and altIcon for every services and use this structure for all services

Reply:
<img width="2053" alt="Screenshot 2024-02-25 at 22 15 00" src="https://github.com/MABPEngineering/brand-website-frontend/assets/66686069/7cfae4cf-6302-40f8-8b36-10660d1231cd">
As you can see here, I can structure better the translation files. But, I would need to refactor other files. For the ServiceCard I'm okay, I needed to save the name and title on a variable so the translation understands. But I'm not happy with the services.json. At the end, this file is connecting with the translation files with the component. So now I needed to change it like title = title (this second "title" it's the one we can see on the translation file). That means that we will end up with a services.json db that has title, description and altIcon being = to the same naming. Are you okay with that?